google/auth/pluggable.py

```diff
@@ -116,6 +117,9 @@ def __init__(
         self._credential_source_executable_timeout_millis = self._credential_source_executable.get(
             "timeout_millis"
         )
+        self._credential_source_executable_interactive_timeout_millis = self._credential_source_executable.get(
+            "interactive_timeout_millis"
```

Comment:
should there be a check that an output file has been provided if interactive timeout millis is present? Check could also happen in the retrieve_subject_token method, just think we need a clear error message if someone tries to run interactive mode with no output file.

Reply:
Should be in `retrieve_subject_token` since we can use both interactive mode and non-interactive mode for a config.
Then we can only check it when actually need to use the output file which is the retrieve_subject_token.